app/views/decks/_deck_form.html.erb

```diff
@@ -18,5 +18,9 @@
     <%= f.label :status %>
     <%= f.select(:status, Deck.statuses.keys.map {|status| [status.titleize, status]}, {}, {:class => 'form-control'}) %>
```

Comment:
did i write this? it needs same as https://github.com/eruditionapp/eruditionapp/pull/29/files#r59316575 i assume you got the pattern from my bad code ;)